src/components/CandidateForm.jsx

```diff
@@ -37,25 +48,42 @@ const CandidateForm = ({ fields, onSubmit }) => {
     onSubmit(obj);
   };
 
-  if (!values) return <Loader></Loader>;
+  if (!values || !technologies) return <Loader></Loader>;
 
   return (
     <>
       <form onSubmit={formHandler} className={styles.form}>
-        {values.map((field) => (
+        {values.map((field) => {
```

Comment:
Here you could simply and just use a ternary.
`field.name !== "technologies" ? <div>...</div> : <Select/>`